pages/_app.tsx

```diff
@@ -160,7 +160,8 @@ class Root extends App<
       throw err;
     }
 
-    const pageProps = await Component.getInitialProps({ ...ctx, egoJwt });
+    const pageProps =
+      (Component.getInitialProps && (await Component.getInitialProps({ ...ctx, egoJwt }))) || {};
```

Comment:
Thinking a ternary may make that more readable? I find the && is better used in short-circuits (where you don't expect a fallback)
```suggestion
      Component.getInitialProps
        ? await Component.getInitialProps({ ...ctx, egoJwt })
        : {};
```